Dockerfile

```diff
@@ -1,15 +1,3 @@
-FROM node AS builder
-
-WORKDIR /app
-COPY package.json package-lock.json ./
-RUN npm install
-
-COPY . ./
-RUN ng build --prod
-
 FROM nginx
-COPY nginx.config /etc/nginx/conf.d/default.conf
-COPY --from=builder /app/dist/RevatureShopFrontend /usr/share/nginx/html
-
-
-
+COPY /dist/RevatureShopFrontend /usr/share/nginx/html
+COPY nginx.conf  /etc/nginx/conf.d/default.conf
```

Comment:
this breaks the ability to use docker compose to as easily run the app in containers, since it relies on manual steps before the docker build can succeed. on the other hand, it's not all that useful to run the frontend in a container.